contrib/examples/actions/workflows/orquesta-task-retry-exhausted.yaml

```diff
@@ -0,0 +1,35 @@
+version: 1.0
+
+description: >
+  A sample workflow that illustrates the use of task retry to rerun task on error. In this example,
+  the workflow will loop until the file /tmp/done exists. A parallel task will wait for some time
+  before creating the file. When completed, /tmp/done will be deleted.
```

Comment:
Description should probably say that this workflow will demonstrate what happens when maximum retry count is reached / exhausted.

Reply:
Done